cmd/kubeadm/app/cmd/config_test.go

```diff
@@ -384,7 +384,7 @@ func TestImagesPull(t *testing.T) {
 func TestMigrate(t *testing.T) {
 	cfg := []byte(dedent.Dedent(`
 		# This is intentionally testing an old API version. Sometimes this may be the latest version (if no old configs are supported).
-		apiVersion: kubeadm.k8s.io/v1beta1
+		apiVersion: kubeadm.k8s.io/v1beta3
```

Comment:
Should this be v1beta2 given the comment above (this is intentionally testing old API versions...)?

Reply:
yes, i think it should be v1beta2.